src/main/java/redis/embedded/core/ExecutableProvider.java

```diff
@@ -12,11 +12,16 @@
 
 public interface ExecutableProvider {
 
+    /**
+     * A system property that allows to configure a custom path for redis-server
+     */
+    String CUSTOM_PATH = System.getProperty("com.github.codemonstur.embedded-redis");
```

Comment:
I don't like this approach. I rewrote the ExecutableProvider a little to make it easier to configure something custom. So if someone wants to use a property they can do so:
```java
final File executable = new File(System.getProperty("pick.something"));
final ExecutableProvider provider = () -> executable;
```